### DISCUSSION TITLE
Trame serving out geometry for viewing via WebGPU ?

### DISCUSSION THREAD

**Discussion by nyue** [Kitware/trame#653] [Q&A]
From reading various google results, WebGPU via Trame may still be a work-in-progress.

Is there any Trame example code/setup I can try out to showcase Trame + WebGPU ?

Kind regards

**Comment by jourdain** [comment]
Not right now, you will get it for free once VTK/C++ has a fully implemented WebGPU backend. This should happen within 2025 for geometry rendering.

Also once available, if you are using trame-vtklocal, the only thing needed will be a `pip install -U vtk` to get WebGPU.
